File: C/Code Snippetts/factorial.c
/*program to print the factorial of a number*/
#include<stdio.h>
void main()
{
    int a,i,f=1;
    printf("ENTER A NUMBER: ");
    scanf("%d",&a);
    for(i=1;i<=a;i++)
    {
        f*=i;
    }
    printf("FACTORIAL=%d",f);
}

File: Python/Code Snippets/FACTORIAL.py
#TO FIND THE FACTORIAL OF A NUMBER
#AUTHOR: ASWIN P
#DATE: 5/7/2022

#FUNCTION
def factorial(n):
    s=1
    for i in range(1,n+1):
            s*=i
    return s

#PROGRAM
n=int(input("ENTER A NUMBER: "))
a=factorial(n)
print("FACTORIAL OF",n,"=",a)